src/content_sao.h

```diff
@@ -377,15 +435,20 @@ class PlayerSAO : public UnitSAO
 	f32 m_fov = 0.0f;
 	s16 m_wanted_range = 0.0f;
 
+	std::unordered_map<std::string, PhysicsModifier> m_physics_modifiers;
+	PhysicsModifier physics_modifier; // Cache for calculatePhysicsModifier
+	bool physics_modifier_dirty = true;
```

Comment:
Private members -> `m_physics_modifier` and `m_physics_modifier_dirty`

Reply:
I thought that we weren't using this convention anymore

Reply:
Nothing changed. https://dev.minetest.net/Code_style_guidelines#Miscellaneous

Reply:
You're right, I remembered a conversation incorrectly.